test/integration/fixtures/cloudconfig.tpl

```diff
@@ -0,0 +1,4 @@
+#cloud-config
+package_upgrade: true
+runcmd:
+ - echo "Created by Azure terraform-vmss-cloudinit module." | sudo dd of=/tmp/terraformtest &> /dev/null
```

Comment:
What's the reason we went with `dd` instead of just `>` to `/tmp/terraformtest`? I know we had to use `dd` previously, but I don't see why cloud-init shouldn't be able to handle the typical way to dump output into a file. This is the integration test artifact, though, so should be fine for its purpose.

Reply:
No reason - just I know it worked... Open to remove the dd, but if we think that's a safer default I'd say let's leave it.